serving/resource_model_serving.go

```diff
@@ -55,13 +83,26 @@ func ResourceModelServing() common.Resource {
 		},
 		Read: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
 			w, err := c.WorkspaceClient()
+			var sOrig serving.ServingEndpointDetailed
+			common.DataToStructPointer(d, s, &sOrig)
 			if err != nil {
 				return err
 			}
 			endpoint, err := w.ServingEndpoints.GetByName(ctx, d.Id())
 			if err != nil {
 				return err
 			}
+			if d.IsNewResource() || sOrig.Config == nil {
```

Comment:
Is `d.IsNewResource()` check needed? Even if it returned true, wouldn't we still want to apply the same logic as the else case?